contrib/devtools/update-translations.py

```diff
@@ -26,7 +26,7 @@
 # Name of transifex tool
 TX = 'tx'
 # Name of source language file
-SOURCE_LANG = 'bitcoin_en.ts'
+SOURCE_LANG = 'xpchain_en.ts'
```

Comment:
Qt .ts file name isn't replace to xpchain name yet.

Reply:
fixed